src/pages/registration/description_tab/FundingsField.tsx

```diff
@@ -1,218 +1,234 @@
-                              setFieldValue(baseFieldName, nfrFunding);
-                            }
-                          }}
-                          required
-                          name={name}
-                          onBlur={onBlur}
-                          errorMessage={touched && !!error ? error : undefined}
-                          data-testid={dataTestId.registrationWizard.description.fundingNfrProjectSearchField}
-                        />
-                      )}
-                    </Field>
-                  ))}
+                    )}
+                  </Field>
 
-                {!hasSelectedNfrSource && hasSelectedSource && (
-                  <>
-                    <Field name={`${baseFieldName}.${SpecificFundingFieldNames.NorwegianLabel}`}>
-                      {({ field, meta: { touched, error } }: FieldProps<string>) => (
+                  {hasSelectedNfrSource &&
+                    (hasSelectedNfrProject ? (
+                      <>
                         <TextField
-                          {...field}
-                          value={field.value ?? ''}
-                          disabled={hasSelectedNfrSource}
+                          value={getLanguageString(funding.labels)}
+                          disabled
                           label={t('registration.description.funding.project')}
                           fullWidth
                           variant="filled"
                           multiline
-                          required={!hasSelectedNfrSource}
-                          error={touched && !!error}
-                          helperText={touched && !!error ? error : undefined}
                           data-testid={dataTestId.registrationWizard.description.fundingProjectField}
                         />
-                      )}
-                    </Field>
-                    <Field name={`${baseFieldName}.${SpecificFundingFieldNames.Identifier}`}>
-                      {({ field }: FieldProps<string>) => (
                         <TextField
-                          {...field}
-                          value={field.value ?? ''}
+                          value={funding.identifier}
                           disabled={hasSelectedNfrSource}
                           label={t('common.id')}
                           fullWidth
                           variant="filled"
                           data-testid={dataTestId.registrationWizard.description.fundingIdField}
                         />
-                      )}
-                    </Field>
+                        {funding.identifier && (
+                          <Button
+                            variant="outlined"
+                            endIcon={<OpenInNewIcon />}
+                            href={getNfrProjectUrl(funding.identifier)}
+                            target="_blank"
+                            data-testid={dataTestId.registrationWizard.description.fundingLinkButton}
```

Comment:
data-testid for funding burde kanskje flyttes til annen plass enn registrationWIzard, da det er samme som brukes for project også

Reply:
la gå..